ui/desktop/tests/e2e/authentication.spec.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef */
+const { test } = require('./fixtures/baseTest');
+const LoginPage = require('./pages/loginPage');
+const { expect } = require('@playwright/test');
+
+/* eslint-disable no-undef */
+test.describe('user/password authentication tests', async () => {
+  test('Authenticates and signs out', async ({
+    electronPage,
```

Comment:
Question for learning:
Looking through playwright docs for test and the [page](https://playwright.dev/docs/api/class-fixtures#fixtures-page). Are we calling this electronPage instead of page for any particular reason?

Reply:
Yup good question, it might be useful to provide some context as to what these fixtures even are. The `page` fixture you're referencing to is a helper to establish and setup a browser window (aka page). That's great for something like admin UI where we do use a browser. 

Unfortunately, the desktop client is an electron app so opening a browser window isn't quite what we want, we need something to actually open electron itself and then get a handle on the browser window _inside_ electron. I could've actually overriden the default `page` and have `electronPage` be called just `page` since I'm not aware of us needing to ever use it but I also didn't want to confuse people nor have people not realize it's actually using electron.  

`electronPage` comes from our own fixture to handle this setup, take a look at the code [here](https://github.com/hashicorp/boundary-ui/blob/5e9a706d7acbc7ca74221b983c842be9a1c294fe/ui/desktop/tests/e2e/fixtures/electronTest.js#L127). For context, look at how we [previously did it](https://github.com/hashicorp/boundary-ui/blob/219eba67bc36df4efe062809ff41b5e5bb96d114/ui/desktop/tests/end2end/authentication.spec.js#L43) and how we needed `beforeEach` hooks everytime to set up electron.

Reply:
oooh ok that makes sense. I see the connection now and why we are calling it electronPage. I would agree that the naming fits. Thanks for pointing me in the right direction, I didn't notice that the first pass.